core/client/fs/src/main/java/alluxio/client/block/stream/DataReader.java

```diff
@@ -22,14 +22,21 @@
 public interface DataReader extends Closeable {
 
   /**
-   * The data reader type.
+   * The data reader type represents different ways to read data.
    */
   enum DataReaderType {
```

Comment:
Which one applies for domain socket? If domain socket and short circuit both use `SHORT_CIRCUIT` maybe we split them or rename?

Reply:
Domain socket using GrpcDataReader. I don't feel I have enough knowledge to do domain socket related changes. The reader type here refects the data reader names/classes that are using.